backend-shared/db/utils.go

```diff
@@ -458,8 +458,10 @@ func SetupForTestingDBGinkgo() error {
 	for _, managedEnvironment := range managedEnvironments {
 		if strings.HasPrefix(managedEnvironment.Managedenvironment_id, "test-") {
 			rowsAffected, err := dbq.DeleteManagedEnvironmentById(ctx, managedEnvironment.Managedenvironment_id)
-			Expect(rowsAffected).Should(Equal(1))
 			Expect(err).To(BeNil())
+			if err == nil {
```

Comment:
Is this `if` check required ? if err was not nil, it would have already failed in the previous line. `Expect(err).To(BeNil())`

Reply:
Not required. I've removed it.

Reply:
Thanks. LGTM